src/utils/AxiosInstance.ts

```diff
@@ -5,10 +5,13 @@ import * as Sentry from '@sentry/browser'
 const instance = axios.create()
 
 instance.interceptors.request.use(
-  config => {
+  request => {
+    if (request.url?.startsWith('https://minio')) {
```

Comment:
Adding the authorization header was causing errors in the newer version of minio. 

Imran will take care of `business-create-ui` + `business-filings-ui`

we don't need to feature flag I believe it's backward compatible